### DISCUSSION TITLE
Get data from hidden columns

### DISCUSSION THREAD

**Discussion by BartoszKosmider** [ghiscoding/Angular-Slickgrid#1006] [Q&A]
Hey, 
Is there any posibility to get data from hidden columns? I would like to use formatter to display in one column values from others. I would like to leave option to hide columns unremain.
getData() returns only data from columns that are visible and I don't see any other method to do that.
Thanks in advance.

**Comment by BartoszKosmider** [comment]
EDIT: grid.getData().getItems()

**Comment by ghiscoding** [comment]
Perhaps I don't understand your question but you already have all the data since you are the person who is providing that said data to the Angular-Slickgrid `[dataset]`

```html
<angular-slickgrid gridId="grid4"
      [columnDefinitions]="columnDefinitions"
      [gridOptions]="gridOptions"
      [dataset]="dataset"
</angular-slickgrid>
```
and please ask such questions on Stack Overflow which is a better platform for Q&A
